synfig-studio/src/gui/app.cpp

```diff
@@ -1253,6 +1253,9 @@ DEFINE_ACTION("keyframe-properties", _("Properties"));
 "		<menuitem action='seek-next-second'/>"
 "		<menuitem action='seek-begin'/>"
 "		<menuitem action='seek-end'/>"
+"		<menuitem action='canvas-zoom-in'/>"
+"		<menuitem action='canvas-zoom-out'/>"
+"		<menuitem action='canvas-zoom-fit'/>"
```

Comment:
I'd prefer those menu items under a new action **menu-view** instead of appending them to an existing **menu-navigation**. Something like this:
```c++
    hidden_ui_info_menu +=
"   <menu action='menu-view'>"
"		<menuitem action='canvas-zoom-in-2'/>"
"		<menuitem action='canvas-zoom-out-2'/>"
"		<menuitem action='canvas-zoom-fit-2'/>"
"	</menu>";
```

Reply:
Okay I will do this. Thanks :)

Reply:
Resolved :)